lib/model/model.go

```diff
@@ -110,6 +110,7 @@ type Model interface {
 	FolderStatistics() (map[string]stats.FolderStatistics, error)
 	UsageReportingStats(report *contract.Report, version int, preview bool)
 	ConnectedTo(remoteID protocol.DeviceID) bool
+	IsFolderPausedOnDevice(folder string, device protocol.DeviceID) bool
```

Comment:
This is the cause of all the build failures; no need to add it to the interface since it's not used outside the package.

Reply:
Thanks! have removed from model and simplified into processNeeded availability check.